### DISCUSSION TITLE
Workout Problems

### DISCUSSION THREAD

**Discussion by Cotrox** [zepp-health/zeppos-forum#101] [Developer Feedback]
I want to let you know some problems i discovered when i use zeppOS for my workout:

- If you start exercise and want to 'rest' before starting a new set, you have to swipe left to right to see the option 'rest'. But if you press the right button the option list you see will not show the option 'rest' and i think can be a nice thing to add it there to make people find it faster.
- Sometimes, when i'm making my set of workout screen become black and i can't see current workout records and info. Then, if i click the right button, sometimes info come back, other time i can also click on options like 'finish'.

- In the weather app there is an horizontal text moviments for too long text but I think you can set a longer width because the box that contains the text is too small compared to the screen.

- I also suggest you to add the double press option on the right button of watches like Amazfit GTS 4. Probably it can be useful if you put an option on the app that user can setup as a shortcut to open a specific app, or to open last opened app. And you can configure it also to make things inside specific app. For exemple if you are making time test and you have to make a time-stamp you can double tap on the timer app to make a time-stamp faster without click the screen.

**Comment by MyoungXUE** [comment]
Hi, these suggestions are very valuable to us and I will send them back to the relevant colleagues, thank you very much!

**Reply by Cotrox** [reply]
Thank you for listen to us always!
